### ISSUE TITLE
Elementor breaks ninja forms on WP 5.6 (PHP 7.4)

### ISSUE THREAD

**Issue by tw1t611** [elementor/elementor#13281]
## Prerequisites
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
Elementor breaks ninja forms on WP 5.6 (PHP 7.4).
The part where you edit the forms is missing.
![image](https://user-images.githubusercontent.com/13353525/101895724-34202b00-3ba8-11eb-9b67-b48973864e33.png)

Error report of /wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5
https://pastebin.com/M7hwYB5R

I did contact the ninja forms team. They directed me to you.
I am using a multi site setup. The problem occures on single site activation and network activation.

## Steps to reproduce
1. Activate ninja forms and elementor
2. Goto ninja form -> any form -> any tab ([Form Fields | Emails & Actions | Advanced])

## Isolating the problem
- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.19.4
	MySQL version: mariadb.org binary distribution v10.5.8
	PHP Version: 7.4.12
	PHP Max Input Vars: 10000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.6
	Site URL: https://staging-arimotors.kinsta.cloud
	Home URL: https://staging-arimotors.kinsta.cloud
	WP Multisite: Yes
	Max Upload Size: 98 MB
	Memory limit: 256Mggkj
	Permalink Structure: /blog/%year%/%monthnum%/%day%/%postname%/
	Language: de-DE
	Timezone: Europe/Berlin
	Admin Email: kontakt@webdasein.net
	Debug Mode: Inactive

== Theme ==
	Name: Ari Elektrotransporter
	Version: 0.1
	Author: Richard Karsten, webdasein UG
	Child Theme: Yes
	Parent Theme Name: GeneratePress
	Parent Theme Version: 3.0.2
	Parent Theme Author: Tom Usborne

== User ==
	Role: 
	WP Profile lang: de_DE
	User Agent: Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:83.0) Gecko/20100101 Firefox/83.0

== Active Plugins ==
	Autoptimize
		Version: 2.8.0
		Author: Frank Goossens (futtta)

	Borlabs Cookie - Cookie Opt-in
		Version: 2.2.13
		Author: Benjamin A. Bornschein, Borlabs

	Classic Editor
		Version: 1.6
		Author: WordPress Contributors

	Custom Post Type UI
		Version: 1.8.1
		Author: WebDevStudios

	Elementor
		Version: 3.0.14
		Author: Elementor.com

	FooBox Image Lightbox
		Version: 2.7.16
		Author: FooPlugins

	GP Premium
		Version: 1.12.2
		Author: Tom Usborne

	Header Footer Code Manager
		Version: 1.1.8
		Author: 99robots

	LeadBooster by Pipedrive
		Version: 1.0.5
		Author: Pipedrive

	MailPoet 3 (New)
		Version: 3.56.0
		Author: MailPoet

	Menu Image
		Version: 2.9.7
		Author: Rui Guerreiro

	Ninja Forms
		Version: 3.4.33
		Author: Saturday Drive

	Ninja Forms - Conditional Logic
		Version: 3.0.28
		Author: The WP Ninjas

	Ninja Forms - File Uploads
		Version: 3.3.10
		Author: The WP Ninjas

	Ninja Forms - Layout & Styles
		Version: 3.0.28
		Author: The WP Ninjas

	Ninja Forms - Multi-Part Forms
		Version: 3.0.26
		Author: The WP Ninjas

	Ninja Forms - Zapier
		Version: 3.0.8
		Author: Fatcat Apps

	Premium Addons for Elementor
		Version: 4.1.0
		Author: Leap13

	SendWP
		Version: 1.0.3
		Author: 

	Seta PDF Mapper
		Version: 1.0.6
		Author: MedienTeam Dresden GmbH

	Smush
		Version: 3.8.2
		Author: WPMU DEV

	Spin360
		Version: 1.2.6
		Author: Maurice

	Widget Context
		Version: 1.3.2
		Author: Kaspars Dambis

	WP Google Maps
		Version: 8.0.29
		Author: WP Google Maps

	Yoast SEO
		Version: 15.4
		Author: Team Yoast


== Network Plugins ==
	MaxButtons
		Version: 8.4.1
		Author: Max Foundry

	MultilingualPress
		Version: 3.5.0
		Author: Inpsyde GmbH

	Premium Addons for Elementor
		Version: 4.1.0
		Author: Leap13


== Must-Use Plugins ==
	Kinsta Must-use Plugins
		Version: 2.3.4
		Author: Kinsta Team


== Elements Usage ==
	
	page : 3
		accordion : 1
		button : 8
		divider : 3
		google_maps : 1
		heading : 9
		icon-box : 13
		image : 4
		image-carousel : 1
		spacer : 2
		text-editor : 4
		video : 1
		wp-widget-ninja_forms_widget : 1
	section : 1
		text-editor : 1
	wp-post : 0
		accordion : 18
		button : 73
		divider : 37
		heading : 158
		icon-box : 149
		image : 75
		image-carousel : 41
		image-gallery : 27
		premium-addon-blog : 1
		premium-addon-modal-box : 16
		shortcode : 1
		spacer : 53
		text-editor : 134
		video : 24
		wp-widget-ninja_forms_widget : 24
	wp-page : 45
		accordion : 3
		button : 14
		divider : 8
		heading : 23
		icon-box : 37
		icon-list : 1
		image : 1
		image-carousel : 7
		image-gallery : 4
		premium-addon-modal-box : 1
		spacer : 16
		text-editor : 14
		video : 3
		wp-widget-ninja_forms_widget : 5


== Log ==
	
JS: showing 3 of 3JS: 2019-09-27 11:31:25 [error X 5][https://www.ari-motors.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.7.3:32:132313] Cannot read property \&#039;html\&#039; of undefined 
JS: 2019-10-11 09:38:00 [error X 6][https://www.ari-motors.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.7.4:7:20270] Cannot read property \&#039;noUiSlider\&#039; of undefined 
JS: 2020-02-04 13:06:21 [error X 1][https://www.ari-motors.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.8.5:31:36487] Cannot read property \&#039;0\&#039; of undefined 

Log: showing 20 of 362020-08-01 12:37:24 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.13',
  'to' => '2.9.14',
)]
2020-08-26 19:25:31 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.14',
  'to' => '3.0.2',
)]
2020-08-26 19:25:31 [info] elementor::elementor_updater Started 
2020-08-26 19:25:31 [info] Elementor/Upgrades - _v_3_0_0_move_general_settings_to_kit Start  
2020-08-26 19:25:31 [info] Elementor/Upgrades - _v_3_0_0_move_general_settings_to_kit Finished 
2020-08-26 19:25:32 [info] elementor::elementor_updater Started 
2020-08-26 19:25:32 [info] Elementor/Upgrades - _v_3_0_0_move_default_colors_to_kit Start  
2020-08-26 19:25:32 [notice] System colors are disabled. nothing to do. 
2020-08-26 19:25:32 [info] Elementor/Upgrades - _v_3_0_0_move_default_colors_to_kit Finished 
2020-08-26 19:25:33 [info] elementor::elementor_updater Started 
2020-08-26 19:25:33 [info] Elementor/Upgrades - _v_3_0_0_move_saved_colors_to_kit Start  
2020-08-26 19:25:33 [info] Elementor/Upgrades - _v_3_0_0_move_saved_colors_to_kit Finished 
2020-08-26 19:26:37 [info] elementor::elementor_updater Started 
2020-08-26 19:26:37 [info] Elementor/Upgrades - _v_3_0_0_move_default_typography_to_kit Start  
2020-08-26 19:26:37 [notice] System typography is disabled. nothing to do. 
2020-08-26 19:26:37 [info] Elementor/Upgrades - _v_3_0_0_move_default_typography_to_kit Finished 
2020-08-26 19:26:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.14',
  'to' => '3.0.2',
)]
2020-10-19 10:02:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.2',
  'to' => '3.0.11',
)]
2020-11-17 09:29:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.11',
  'to' => '3.0.13',
)]
2020-11-26 10:52:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.13',
  'to' => '3.0.14',
)]

PHP: showing 4 of 4PHP: 2020-11-24 13:49:25 [warning X 5][/homepages/34/d740732630/htdocs/wordpress/wp-content/plugins/elementor/core/files/base.php::180] file_put_contents(/homepages/34/d740732630/htdocs/wordpress/wp-content/uploads/elementor/css/post-4357.css): failed to open stream: Disk quota exceeded [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-11-24 13:50:53 [warning X 5][/homepages/34/d740732630/htdocs/wordpress/wp-content/plugins/elementor/core/files/base.php::180] file_put_contents(/homepages/34/d740732630/htdocs/wordpress/wp-content/uploads/elementor/css/post-3924.css): failed to open stream: Disk quota exceeded [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-11-24 14:37:15 [warning X 3][/homepages/34/d740732630/htdocs/wordpress/wp-content/plugins/elementor/core/files/base.php::180] file_put_contents(/homepages/34/d740732630/htdocs/wordpress/wp-content/uploads/elementor/css/post-4378.css): failed to open stream: Disk quota exceeded [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-11-24 14:43:12 [warning X 2][/homepages/34/d740732630/htdocs/wordpress/wp-content/plugins/elementor/core/files/base.php::180] file_put_contents(/homepages/34/d740732630/htdocs/wordpress/wp-content/uploads/elementor/css/post-4385.css): failed to open stream: Disk quota exceeded [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
```
</details>

**Comment by VenkmanNL** [comment]
I've got the same problem with ninja forms. 
See also #13273

**Comment by shilo-ey** [comment]
Thanks for reporting.

We will address this issue soon and release a patch.

Please follow this thread for updates.

Thanks!

**Comment by RubenAlinoa** [comment]
Same issue ;-)

**Comment by Eigilak** [comment]
I also got the issue :)

**Comment by shilo-ey** [comment]
We are currently working on a fix.

This issue origins from the upgrade of jQuery version build in WP 5.6

To avoid this issue for the meanwhile please use the following plugin:
https://wordpress.org/plugins/enable-jquery-migrate-helper/

A patch will release in the upcoming days.

Thanks

**Comment by VenkmanNL** [comment]
The jquery-migrate-helper doesn't seem to do the trick for Ninja forms problem

**Comment by davidmesguerra** [comment]
Any other alternatives to fix the issue? the JQuery Migrate Helper plugin doesn't fix it

**Comment by huszonot** [comment]
Same problem here!

**Comment by luisalpaba** [comment]
Temporary solution provided by Ninja Forms: Our developers are currently investigating a conflict with an Elementor script which is currently affecting the ability to edit forms.  Our engineering team is investigating the conflict and reaching out to the Elementor team for insights.
To alleviate the issue short-term, please temporarily roll WordPress back to version 5.5.3.
Alternatively, you can try using the "Enable jQuery Migrate Helper" plugin:
https://wordpress.org/plugins/enable-jquery-migrate-helper/
After installing this, navigate to Tools > jQuery Migrate and set the jQuery version to "Legacy 1.12.4-wp".

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.15

Feel free to update

Thanks!
